test/memorize_test.dart

```diff
@@ -0,0 +1,47 @@
+import 'package:memoize/memoize.dart';
+import 'package:test/test.dart';
+
+void main() {
```

Comment:
Add test for complex object, object with custom hashcode func, add test for initial null